### PR TITLE
New UI and Navbar

### PR DESCRIPTION
**Pull request by alperendnc**
# Todo List Application

This repository contains a simple Todo List application that allows users to track their tasks. The application provides essential functionalities such as adding, editing, and deleting tasks.

## Pull Request: New Navbar and User Interface Updates

This pull request includes a series of significant changes and improvements aimed at enhancing the user interface of the Todo List application and improving user experience.

### Changes

1. **New Navbar Design**:
   - A modern and user-friendly navbar has been added.
   - Restructured to help users navigate quickly within the application.
   - New links like "Search" and "Upcoming Today" provide easy access to essential information.

2. **User Interface Enhancements**:
   - Overall aesthetics improved with updates to background color and typography.
   - The task entry area has been made more prominent; users can click on "Add Task" to reveal a new input field for entering tasks.

3. **Code Clean-Up and Refactoring**:
   - Removed unnecessary CSS and JavaScript code.
   - Improved readability and maintainability of the code.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,80 +1,74 @@
...
   display: flex;
+  flex-direction: column;
   justify-content: space-between;
-  width: 100%;
-  margin: 10px;
+  width: 95%;
+  margin: 10px auto;
   padding: 20px;
-  max-width: 1400px;
   gap: 20px;
```
Comment (ucemrecan) [inline src/App.css:19]: gap calisiyor mu burada? space-between saglar gibi onu

File: src/components/TodoForm.js [hunk 2]
```diff
@@ -28,26 +21,16 @@ const TodoForm = () => {
 
   const handleSubmit = (e) => {
     e.preventDefault();
+    if (task.task.trim() === "") {
```
Comment (ucemrecan) [inline src/components/TodoForm.js:24]: Burasi sikintili. 
- task.task isimlendirmesi kötü
- neden boyle yapildi? Task girilmediyse kullanici submit edemesin, buraya girene kadar

File: src/components/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    <nav className="navbar">
+      <ul>
+        <li>
+          <a href="/">
+            <FontAwesomeIcon className="nav-icon" icon={faSearch} />
+            Ara
+          </a>
+        </li>
+        <li>
+          <a href="/bugun">
```
Comment (ucemrecan) [inline src/components/Navbar.js:19]: filtrelemeyi router ile yapmayi dusunuyorsan. bu sekilde olmamali, sayfa gecisine gerek yok tek sayfada tarihe gore filtrelemelisin

**Review by ucemrecan — CHANGES_REQUESTED** [review]
Düzeltilmesi gereken yerler var

File: src/App.css (hunk 1 above)
Comment (alperendnc) [reply src/App.css:19]: tam tersi oluyor abi aksine space-between calismiyor ancak gap calisiyor

File: src/components/Navbar.js (hunk 3 above)
Comment (alperendnc) [reply src/components/Navbar.js:19]: navbari daha cok gelistirmeyi dusunuyorum abi suan bundan dolayi sayfa gecisini kullandim ama filtreleme icin haklisin sayfa gecisini kaldiralim

File: src/components/TodoForm.js (hunk 2 above)
Comment (alperendnc) [reply src/components/TodoForm.js:24]: task trimi direkt kaldirdim artik task girmeden submit butonu gozukmuyor

**Pull request merged**